main/jlp/widgets/widget_factory.cpp

```
@@ -2480,6 +2497,294 @@ lv_obj_t* build_voice(BuildCtx& ctx, JsonObjectConst spec, std::string* err) {
+      c->watch_timer = cockpit_hal::ui::every(1000, [c]() { stream_watch_tick(c); });
+    }
+  } else {
+    if (c->watch_timer) {
+      cockpit_hal::ui::cancel(c->watch_timer);
+      c->watch_timer = 0;
+    }
+    stream_do_stop(c);
+  }
+}
+
+void stream_touch_send(StreamCtx* c, uint16_t x, uint16_t y, uint8_t type) {
```

Clang (14.0.6)

[warning] parameter name 'c' is too short, expected at least 3 characters

(readability-identifier-length)